.gitignore

```diff
@@ -93,3 +93,13 @@ docs/python_api/
 docs/conf.py
 docs/Doxyfile
 docs/getting_started.rst
+docs/tutorial/visualization/!temp_scratchpad.rst
```

Comment:
@yxlao `.gitignore` is, in this case, my computer's specific file. **Of course, it could be deleted.** I filled it just for convenience of focusing on only files I have to push via GitHub Desktop.